components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/rabbitmq/RabbitMQInjectHandler.java

```diff
@@ -47,10 +48,23 @@
     private String onErrorSeq;
     private boolean sequential;
     private SynapseEnvironment synapseEnvironment;
+    private SequenceMediator seq;
 
     public RabbitMQInjectHandler(String injectingSeq, String onErrorSeq, boolean sequential,
-                                 SynapseEnvironment synapseEnvironment) {
+                                 SynapseEnvironment synapseEnvironment) throws SynapseException {
         this.injectingSeq = injectingSeq;
+        if (injectingSeq == null || injectingSeq.equals("")) {
+            String msg = "Injecting Sequence name is not specified.";
+            log.error(msg);
+            throw new SynapseException(msg);
+        }
+        seq = (SequenceMediator) synapseEnvironment.getSynapseConfiguration().getSequence(injectingSeq);
+        if (seq == null) {
+            throw new SynapseException("Sequence Mediator is null.");
```

Comment:
The log should be "specified injecting sequence: <name>" is invalid.

Reply:
Done in b459fdfdf15ed24137e23f5b3a394f2683be6223